### ISSUE TITLE
Search bar opens up voice assistant instead

### ISSUE THREAD

**Issue by abigdeel** [KieronQuinn/DiscoverKiller#9]
CrDroid 8.2 (Android 12) on a OP7 Pro.

With DiscoverKiller enabled, the Pixel Launcher Google search bar doesn't function properly.

Tapping anywhere on the search bar opens up the voice assistant instead. With it disabled, the search bar works fine. Then the search, voice assistant, and Lens buttons work as they should.

With snapshot leaving soon though, I wonder what that will mean for this module. Funny cause I finally just installed a custom ROM again after years of running stock. Last time was my Nexus 4 I think.

**Comment by KieronQuinn** [comment]
This appears to be a Pixel Launcher bug, I had it before I reset a couple of weeks ago and then it's been fine since. I've also seen it reported by others who aren't even rooted - let alone using Discover Killer. Whether enabling and disabling DK is resetting something, I don't know.

**Comment by KieronQuinn** [comment]
As for what it will do once Snapshot is dead, I'm leaning towards the options being "app" (as it is now), and a third party overlay, if one is installed. 

There **are** third party overlays that may provide a good replacement, such as https://github.com/iTaysonLab/HomeFeeder, Discover Killer could redirect the official one to allow it on all launchers I think.

**Comment by abigdeel** [comment]
Appreciate the responses.

I would agree that there is a problem with Pixel Launcher itself. Before I installed your module, I noticed that it happened a couple times but restarting the phone took care of it. However now a restart doesn't seem to help, but toggling the module on and off reproduces/fixes the problem every time.

I don't feel like going through another reset so I'll have to leave the module off for now. Unless you think there is a more conservative solution I could try to get it working. Otherwise I'll just suffer through Discover for a little longer. Feel free to close this issue.
